diff --git a/TeX.gitignore b/TeX.gitignore
index e964244..d44aa72 100644
--- a/TeX.gitignore
+++ b/TeX.gitignore
@@ -17,9 +17,10 @@
 *.xdv
 *-converted-to.*
 # these rules might exclude image files for figures etc.
-# *.ps
+# makefile4latex: ignore *.ps *.pdf
+*.ps
 # *.eps
-# *.pdf
+*.pdf
 
 ## Generated if empty string is given at "Please type another file name for output:"
 .pdf
@@ -116,7 +117,8 @@ acs-*.bib
 *.sls
 
 # uncomment this for glossaries-extra (will ignore makeindex's style files!)
-# *.ist
+# makefile4latex: ignore *.ist
+*.ist
 
 # gnuplot
 *.gnuplot
@@ -144,7 +146,8 @@ acs-*.bib
 # knitr
 *-concordance.tex
 # TODO Uncomment the next line if you use knitr and want to ignore its generated tikz files
-# *.tikz
+# makefile4latex: ignore *.tikz
+*.tikz
 *-tikzDictionary
 
 # listings
@@ -298,4 +301,5 @@ TSWLatexianTemp*
 # REVTeX puts footnotes in the bibliography by default, unless the nofootinbib
 # option is specified. Footnotes are the stored in a file with suffix Notes.bib.
 # Uncomment the next line to have this generated file ignored.
-#*Notes.bib
+# makefile4latex: ignore *Notes.bib
+*Notes.bib
